### ISSUE TITLE
Is this project abandoned ?

### ISSUE THREAD

**Issue by Swizz** [dhg/Skeleton#311]
Hi.

I see many opened pull requests and issues.
It raises on my mind this question.

I have one or two ideas to improve some things.

If this project is abandoned. Could we move to another maintener ?

Kind regards.

**Comment by dan1elhughes** [comment]
Last commit from @dhg was 29 Dec 2014 so looks like it. There is a fork [here](https://github.com/SkeletonFramework/SkeletonFramework/) that's being actively maintained.

**Comment by ghost** [comment]
I created a new Version of Skeleton called Skeleton plus.
It was created yesterday so my version will be active.
http://github.com/oltmannsdaniel/skeleton-plus

**Comment by cottsak** [comment]
I think this is the more appropiate link https://github.com/skeleton-framework/skeleton-framework

**Comment by ghost** [comment]
Now is and Sass Version available at 
https://github.com/oltmannsdaniel/skeleton-plus

**Comment by ghost** [comment]
Well, skeleton-plus or skeleton-framework ? It's look like skeleton-plus is more than the original version, not the same "simple" philosophy, i'm right ?
So if i just want a maintained version of skeleton, i'll prefer skeleton-framework ?

**Comment by ghost** [comment]
@Lud-up skeleton-plus is the originial Version with some little extra features

**Comment by ghost** [comment]
@oltmannsdaniel  I like skeleton plus for it's responsive menu  thanks :-)

**Comment by naveenlb** [comment]
@oltmannsdaniel  What are the new features in your version? If you can share the tutorial or anything about it, then it would be great!!!

**Comment by naveenlb** [comment]
@oltmannsdaniel It's ok I got it. Thanks. Looks good man. Appreciate it.

http://oltmannsdaniel.github.io/skeleton-plus/

**Comment by Borderliner** [comment]
I've created a black version (very different in style though) of Skeleton, called Meshki. I actively maintain it.

GitHub: https://github.com/Borderliner/Meshki

Showcase: http://meshki.borderliner.ir/

**Comment by tiaanduplessis** [comment]
I created a boilerplate based heavily on skeleton using [Flexbox grid](http://flexboxgrid.com/) and [sanitize.css](https://github.com/jonathantneal/sanitize.css).
U can find it here: https://github.com/mightyCrow/barebones-css
Would appreciate any feedback.

**Comment by tworkman512** [comment]
Sad this seems to be abandoned :( 

I've used this grid system a while back because I liked how dead simple it is. Glad to see a version of it with https://github.com/skeleton-framework/skeleton-framework. Had no idea that was out there, so thanks! 

It's nice to have such a simple grid boilerplate to use in projects that doesn't include a bunch of extras.

**Comment by Bigben83** [comment]
your better off just using this framework its in active dev and very similar
https://github.com/Borderliner/Meshki

**Comment by tworkman512** [comment]
@Bigben83 thanks for sharing! Will check that repo out.

**Comment by Swizz** [comment]
@tworkman512 If you are just looking for a grid system what about 
[rohitkrai03/pills ](/rohitkrai03/pills) ?

**Comment by Borderliner** [comment]
@Swizz Grid Systems are really useful for starting a new UI framework. But I guess people use [Skeleton](https://github.com/dhg/Skeleton) because it's small, simple and ready to use. That's the exact same goal of [Meshki](https://github.com/Borderliner/Meshki), though it provides more useful components. There's also [Skeleton Framework](https://github.com/skeleton-framework/skeleton-framework/) which is developed independently.

**Comment by Swizz** [comment]
@Borderliner I understand the Meshki goals and I appreciate the new common objective of minimal size dependencies carried out by Riot, Umbrella, and Meshki for example.

Few month before, I was on the way of "all features included" HTML/CSS/JS frameworks.
But today I moved into more composable and light solutions.
I have totally abandoned framework philosophy in favor of libraries.

For CSS stuffs, I overuse of post-css and lost for example ; No more using of CSS Components Full Frameworks, right now I build all stylesheet "from scratch"

**Comment by jKratzik** [comment]
I am using frameworks myself for many projects (mostly Foundation) but they are bloated with features that I actually never need. A 119 KB minified Bootstrap or even 76 KB minified Foundation CSS is simply too much for building websites.

So I was thinking to build a modular framework that consists of 3 parts: typography - grid - and "components" CSS. Developers could use only typography, typography and grid or all 3 parts - or even include certain components separately.
And Skeleton would be a great starting point just it's a pity that it's not anymore under active development.

@dhg Dave I am very interested to take over maintenance for the framework if you are willing to hand it over.

**Comment by TristanAG** [comment]
@dhg @jKratzik that sounds cool.. what sort of components are you thinking of? I really want to keep this framework alive and active, I love how lightweight and simple it is.

**Comment by 1fabiopereira** [comment]
@jKratzik Currently I keep the Skeleton-Plus that is the same base of the skeleton so that already modularized with sass, lately I walk without time to give maintenance I would like to have a look?

https://github.com/oltmannsdaniel/skeleton-plus

**Comment by naveenlb** [comment]
@dhg , the creator of Skeleton is working for Twitter Bootstrap now. So this project is abandoned. Check out his pic in the link http://themes.getbootstrap.com/

**Comment by ShashankaNataraj** [comment]
Ill just leave this here: https://medium.com/@dhg/dear-skeleton-452f4bb07d69#.o5zujlr5i

**Comment by jimrrchen** [comment]
@ShashankaNataraj 
Based on the medium article, it seems like @dhg abandoned skeleton once and pick it up again. And that was on Dec 9, 2014 (when Skeleton 2.0 launched). But the last commit for skeleton is Dec 29, 2014...so it looks like @dhg's commitment to Skeleton for the second time lasted for......20 days?

Still a great framework nonetheless, I learned a lot by just reading through the code

**Comment by Borderliner** [comment]
Of course. It's very amusing that such a simple code can do so much. Though it has some bugs, and lacks some useful features like navigation bars, footers, has mediocre font, etc.

**Comment by ShashankaNataraj** [comment]
@jimrrchen Yes, its a simple project that does what it does very well. Though Id have liked it more if SCSS splits were available so I could avoid loading parts of skeleton that I dont like and create a custom build... But, great framework nonetheless.

Also anyone looking at this should also look at https://github.com/rohitkrai03/pills if they're looking for a minimalistic CSS grid lib.

**Comment by Borderliner** [comment]
@ShashankaNataraj Nice suggestion about SCSS splits. I'll rewrite [Meshki](https://github.com/Borderliner/Meshki) as soon as possible, so that users can make custom builds. Thank you.

**Comment by EminezArtus** [comment]
Check this out its the basically the same concept but with flex-box http://milligram.io/

**Comment by Borderliner** [comment]
If anybody has a solid knowledge of flex-box, I'd really appreciate if once could help me implement it for [Meshki](https://github.com/Borderliner/Meshki/tree/sass-dev). <3

**Comment by lunarcloud** [comment]
CSS Grid makes even flex-based responsive frameworks obsolete and unnecessary.
Move on. Hopefully there will be frameworks that are just about consistent theming or whatnot and not about the grid.

**Comment by andreobriennz** [comment]
I'm a bit late commenting here, but I created a fork called skeleton-flexbox to replace Skeletons grid with a flexbox grid. Version 3 uses Skeleton with flexbox-grid (http://flexboxgrid.com), which I find very nice to use. 

Feel free to check it out, and would appreciate any feedback: https://github.com/andreobriennz/skeleton-flexbox

I'll also look at splitting it into different SCSS files next week, and possibly add some variables for colors, etc

**Comment by davidjendreizak** [comment]
I have created a fork [here](https://www.github.com/davidjendreizak/contour). I'd like to add cards, badges and more.

**Comment by acahir** [comment]
Just to add yet another fork, I've updated Skeleton to use modern CSS features and kept it dependency free (ie. no SCSS required):

[Barebones](https://github.com/acahir/Barebones)
- CSS Grid replaces 12 columns
- CSS Variables used globally for theme styling
- Maintained backwards legacy support for Skeleton styling but build on top of CSS Grid and CSS Variables

**Comment by Selekkt** [comment]
Take a look at [Skelet.](https://selekkt.dk/skelet/v3/) 
- Minimal and straight-forward CSS grid system utilising descriptive HTML rather than semantic CSS
- Packed with Flexbox utilities
- A bunch of class utilities to speed your workflow 
- CSS Variables for full customisation
- Few basic animations to use right away
- and a lot more
